js/modules/loki_file_server_api.js

```diff
@@ -8,11 +8,49 @@ const LokiAppDotNetAPI = require('./loki_app_dot_net_api');
 const DEVICE_MAPPING_USER_ANNOTATION_TYPE =
   'network.loki.messenger.devicemapping';
 
+// const LOKIFOUNDATION_DEVFILESERVER_PUBKEY =
+//  'BSZiMVxOco/b3sYfaeyiMWv/JnqokxGXkHoclEx8TmZ6';
```

Comment:
What is the key in the comment? Can we remove it?

Reply:
this is for file-dev which we should be using for development but can't use until we deploy another storage-server update

Reply:
I see, I missed the `DEV` part.